manifests/0000_90_kube-apiserver-operator_03_servicemonitor.yaml

```diff
@@ -15,8 +15,7 @@ spec:
     port: https
     scheme: https
     tlsConfig:
-      # TODO where do you mount the service-ca.crt?
-      insecureSkipVerify: true
+      caFile: /etc/prometheus/configmaps/serving-certs-ca-bundle/service-ca.crt
```

Comment:
@sttts this seems to be monitoring team magic we have to follow

Reply:
We're working on allowing people to do secret/configmap key selectors instead, we'll let people know once it's available and tested. This is an unfortunate artifact of early poor API decisions ...